server/api/models/post.js

```diff
@@ -0,0 +1,68 @@
+const db = require("../dbConfig");
+
+class Post {
+  constructor(data) {
+    this.id = data.id;
+    this.title = data.title;
+    this.username = data.username;
+    this.body = data.body;
+  }
+
+  static get all() {
+    return new Promise(async (resolve, reject) => {
+      try {
+        const postData = await db.query("SELECT * FROM post;");
+        console.log(postData);
```

Comment:
I believe this is a leftover bug catcher you may want to erase to perfect your code.